control/src/spi_interface.c

```diff
@@ -0,0 +1,90 @@
+
+
+#include "spi_interface.h"
+
+#include <wiringPiSPI.h>   // only available on Raspberry Pi 
+
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+
+/*************************************************************************************************************************************************
+ * Globals Variables
+ * 
+ * Notes:
+ * Channel is the wiringPi name for the chip select (or chip enable) pin. Both channel 0 and channel 1 will be used.
+ * Will add logic to allow for both later.
+*************************************************************************************************************************************************/
+static const int CHANNEL0 = 0;
+static const int CHANNEL1 = 1;
+
+/*************************************************************************************************************************************************
+ * Function: 
+ * 
+ * Notes:
+ * Channel is the wiringPi name for the chip select (or chip enable) pin. Both channel 0 and channel 1 will be used.
+ * Will add logic to allow for both later.
+*************************************************************************************************************************************************/
+bool spiLogic(char* instruction)
+{
+    bool result;
+
+
+    if( strcmp(instruction, "CLR") == 0 )          // instruction requires 1 byte
+    {
+
+    }
+    else if( trcmp(instruction, "RD") == 0 )       // instruction requires 2-5 bytes
```

Comment:
There's a typo, but I recommend replacing these string variables being passed into the function with #define constants or an enumerated type

Reply:
Did replacement with #define constants